frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.jsx

```diff
@@ -358,6 +355,12 @@ class PivotTable extends Component {
       valueIndexes,
     } = pivoted;
 
+    const { leftHeaderWidths, totalHeaderWidths } = getLeftHeaderWidths({
+      rowIndexes: rowIndexes ?? [],
+      getColumnTitle: idx => this.getColumnTitle(idx),
+      fontFamily: MetabaseSettings.get("application-font") ?? "Lato",
```

Comment:
AFAIR we agreed to avoid using `MetabaseSettings` like that in favor of reading them from the store. In this case, you won't need to put a default "Lato" font here